django/db/backends/base/schema.py

```diff
@@ -36,10 +36,16 @@ def _all_related_fields(model):
 def _related_non_m2m_objects(old_field, new_field):
     # Filter out m2m objects from reverse relations.
     # Return (old_relation, new_relation) tuples.
-    return zip(
+    related_fields = zip(
         (obj for obj in _all_related_fields(old_field.model) if _is_relevant_relation(obj, old_field)),
         (obj for obj in _all_related_fields(new_field.model) if _is_relevant_relation(obj, new_field)),
```

Comment:
I was curious about something when seeing this. Must `_is_relevant_relation()` always give the same result for `new_field` as for `old_field`, in order for the `zip()` to line up? If so, it seems like a simplification could be done in a subsequent PR.

Reply:
Yes, `_is_relevant_relation` should return the same result :thinking: